server_admin/topics/realms/email.adoc

```diff
@@ -6,7 +6,7 @@
 
 .Procedure
 
-. Click *Realm Settings* in the menu.
+. Click *Realm settings* in the menu.
```

Comment:
Do you have a -prod version of this file saved for use with SSO. We will need a version saved of every file being changed or the ongoing SSO releases.